docs/installation.rst

```diff
@@ -2,6 +2,54 @@
 Installation
 ============
 
+--------------
+Docker Install
+--------------
+
+For quick installation, consider installing from our
+`Docker image <https://hub.docker.com/r/eregs/parser/>`_.
+This image includes all of the relevant dependencies, wrapped up in a
+"container" for ease of installation. To run it, you'll need to have Docker
+installed, though the installation instructions for
+`Linux <https://docs.docker.com/linux/step_one/>`_,
+`Mac <https://docs.docker.com/mac/step_one/>`_, and
+`Windows <https://docs.docker.com/windows/step_one/>`_
+are relatively painless.
+
+To run with Docker, there are some nasty configuration details which we'd like
+to hide behind a cleaner interface. Specifically, we want to provide a simple
+mechanism for collecting output, keep a cache around in between executions,
+allow input/output via stdio, and prevent containers from hanging around in
+between executions. To do that, we recommend creating a wrapper script and
+executing the parser through that wrapper.
+
+For Linux and OS X, this looks like:
+
+.. code-block:: bash
+
+  # Make a shell script
+  echo '#!/bin/sh' > eregs
+  # The script will create a directory for the output
+  echo 'mkdir -p output' >> eregs
+  # It'll also create a placeholder local_settings.py, if none exists
+  echo 'touch local_settings.py' >> eregs
+  # The script will wrap the docker command with appropriate flags while
+  # passing in any arguments.
+  # --rm removes the container after execution
+  # -v mounts volumes for cache, output, and copies in the local settings
+  echo 'docker run --rm -it -v eregs-cache:/app/cache -v $PWD/output:/app/output -v $PWD/local_settings.py:/app/code/local_settings.py eregs/parser $@' >> eregs
+  # Make the script executable
+  chmod +x eregs
+
+To execute parsing commands, just run ``path/to/eregs`` in place of ``eregs``
```

Comment:
Sentence does not parse right. Did you mean:
To parse, run this wrapper script, ``path/to/eregs``, instead of ``eregs`` wherever instructed to in the documentation.